tasks/install-on-Darwin.yml

```diff
@@ -13,13 +13,24 @@
     state: latest
     update_homebrew: yes
 
+- name: Configure Buildkite service
+  template:
+    src: "homebrew.mxcl.buildkite-agent.plist.j2"
+    dest: "{{ buildkite_agent_conf_dir[ansible_os_family] }}/homebrew.mxcl.buildkite-agent.plist"
+    owner: "{{ buildkite_agent_username }}"
+    group: staff
+    mode: "0600" # macOS requires 0400 or 0600.
```

Comment:
It would be extra beautiful to cite some docs (and I appreciate Apple's docs are hard to search and sometimes contradictory; this is not a blocking comment if you can't find a decent citation!).